tests/client/test_linalg.py

```diff
@@ -108,19 +108,21 @@ def testPLU(self):
             all([np.sum(p[i, :]) == np.sum(p[:, i]) == 1.0 for i in range(p.shape[0])]),
         ])
 
-    @unittest.skip
+    #@unittest.skip
```

Comment:
you should remove this line

Reply:
Yes, of course, I will make a change in the next PR